js/interface.js

```diff
@@ -838,6 +1036,52 @@ $('.file-manager-wrapper')
   })
   .on('hide.bs.collapse', '.panel-collapse', function() {
     $(this).siblings('.panel-heading').find('.fa').removeClass('rotate');
+  })
+  .on('keyup', '.search-term', _.debounce(function(){
+    var term = $searchTerm.val();
+
+    if (!term) {
+      disableSearchState();
+      backToLastFolderBeforeSearch();
+      return;
+    }
+
+    removeSelection();
+    hideSideActions();
+
+    var type = $searchType.val();
+    search(type, term);
```

Comment:
doesn't look like `.catch` is handled with an error

Reply:
Added catch block